### PR TITLE
remove me from copyright notice

### PR DESCRIPTION
**Pull request by marcoow**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2013-2017 simplabs GmbH/Marco Otte-Witte and contributors
+Copyright (c) 2013-2017 simplabs GmbH and contributors
```
Comment (jessica-jordan) [inline LICENSE]: Could we extend the copyright notice until `2018` in the same PR or would you like to update that in a follow-up?

**Comment by marcoow** [comment]
@jessica-jordan: extension to 2018 is done in #1508

**Comment by jessica-jordan** [comment]
@marcoow Isn't the change in #1508 only the part of the docs, whereas this is an edit to the `LICENSE.md`? 🤔 
https://github.com/simplabs/ember-simple-auth/pull/1508/files

**Comment by marcoow** [comment]
Ah sorry, I meant #1506 but that actually went wrong somehow. I'll create a new PR for that.

**Comment by marcoow** [comment]
@jessica-jordan: good to go?

**Pull request merged**